pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1216 @@
+#  *  Notes on language style *
+#  
+#	1. 		When not explicitely using "he, him, her, etc." or situations in which the context implies only men or women, most "group" terms are used to inflect either gender.
+#		This follows what I interpret as a continuation of the modern trend toward encompassing both genders into previously "male" or "female" terms. 
+#		For example, "Lads" equates to a group of men, women, or men and women in the same way as "guys" or "dudes". It does not strictly imply males unless contextually specified. 
+#		My intent is to provide thematic variability and allow conversations to flow naturally where NPCs would normally use the player character's gender 
+#		(which the game does not know) without always using implied addresses or the handful of neutral addresses we currently have (you, <first>, Captain).
+#
+#	
+# # # # #  NOTES ON SYNTAX  # # # # #
+#
+#	1. Conversations
+#		a. "meetupX" is a special label I use to keep conversations organized. These labels indicate the next spot where conversation threads meet back up, regardless of chosen options. They are numbered sequentially for ease-of-use.
+#		b. You may notice "goto" markers just prior to the dialogue label they are meant to be paired with. This is to cut down on mistakes during editing / additions to conversations.
+#		c. "branch" markers are linked to "branchX" labels to keep track of branch labels vs. regular dialogue labels vs. meetup labels.
+#		d. "question" is a special label I use for certain missions where the player can ask a series of questions, but eventually has to say "yes" or "no". These are single-line recursive; they take up less space and do not visibly repeat in-game.
+
+
+
+# * * * * * * * * * *  "INTRO"  * * * * * * * * * *
+mission "Etarip Intro"
+	invisible
+	landing
+	
+	to offer
+		#random < 5
+		"year" == 3013

```

Comment:
Oh nevermind, I think you just put these condition because you commented the random. In the "final" form of the PR (ie just before merging), only the random part will remain right ?

Reply:
You should put it in the final form, the people likely to be reviewing this are capable of making the required changes if they want to force a trigger.

Reply:
@jafdy Noted; "random" has been uncommented.